keps/sig-release/3000-artifact-distribution/README.md

```diff
@@ -0,0 +1,135 @@
+# KEP 3000: Artifact Distribution Policy
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [SIG Release - Image Promotion](#sig-release---image-promotion)
+    - [Cloud Customer - Installing K8s via kubeadm](#cloud-customer---installing-k8s-via-kubeadm)
+  - [Notes/Constraints/Caveats](#notesconstraintscaveats)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Artifact Promotion](#artifact-promotion)
+    - [Policy](#policy)
+    - [Process](#process)
+  - [Artifact Distribution](#artifact-distribution)
+    - [Policy](#policy-1)
+    - [Process](#process-1)
+- [Alternatives / Background](#alternatives--background)
+  - [How much is this going to save us?](#how-much-is-this-going-to-save-us)
+- [Infrastructure Needed](#infrastructure-needed)
+- [Hack on this doc](#hack-on-this-doc)
+<!-- /toc -->
+
+## Summary
+
+The container images and release binaries produced by our community need a clear path to be hosted by multiple service/cloud providers.
+
+The global community should be routed to the appropriate mirror for their country or cloud provider to ensure cost effective worldwide access.
+
+This KEP should cover the policy and distribution mechanisms we will put in place to allow creating a globally distributed, multi-cloud and country solution.
+
+## Motivation
+
+Currently we push to a single provider, and distributing to the rest of community comes at great cost nearing $150k/month (mostly egress) in donations.
+
+Additionally, some of our community members are unable to access the official release artifacts due to country level firewalls that do not them connect to Google services.
+
+Ideally we can dramatically reduce cost and allow everyone in the world to download the artifacts released by our community.
+
+### Goals
+
+A policy and procedure for use by SIG Release to promote container images and release binaries to multiple registries and mirrors.
```

Comment:
I think we should break this into 2 major phases:

1) Container images
2) Other artifacts

We may even want to break it to 2 KEPs so we can "finish" one.

Reply:
Good thought! This KEP just focuses on container images now.